### PR TITLE
Add metrics traces use case to Data Prepper

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Add metrics traces use case to Data Prepper

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@dlvenable Please review this use case PR at your availability. Thanks, Melissa

**Review by dlvenable — APPROVED** [review]

File: _data-prepper/common-use-cases/metrics-traces.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+title: Deriving metrics from traces
+parent: Common use cases
+nav_order: 60
+---
+
+# Deriving metrics from traces
+
+You can use Data Prepper to derive metrics from OpenTelemetry traces. The following example pipeline receives incoming traces and extracts a metric called `durationInNanos`, aggregated over a tumbling window of 30 seconds. It then derives a histogram from the incoming traces.
+
+The pipeline contains the following multiple pipelines:
```
Comment (natebower) [inline _data-prepper/common-use-cases/metrics-traces.md:12]: Do we mean "The pipeline contains the following sub-pipelines"? "following multiple" doesn't work here.

File: _data-prepper/common-use-cases/metrics-traces.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- `entry-pipeline` -– Receives trace data from the OpenTelemetry collector and forwards it to the `trace_to_metrics_pipeline` pipeline.
```
Comment (natebower) [inline _data-prepper/common-use-cases/metrics-traces.md:14]:
```suggestion
- `entry-pipeline` – Receives trace data from the OpenTelemetry collector and forwards it to the `trace_to_metrics_pipeline` pipeline.
```

File: _data-prepper/common-use-cases/metrics-traces.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+- `trace-to-metrics-pipeline` –- Receives the trace data from the `entry-pipeline` pipeline, aggregates it, and derives a histogram of `durationInNanos` from the traces based on the value of the `serviceName` field. It then sends the derived metrics to the OpenSearch index called `metrics_for_traces`.
```
Comment (natebower) [inline _data-prepper/common-use-cases/metrics-traces.md:16]:
```suggestion
- `trace-to-metrics-pipeline` – Receives the trace data from the `entry-pipeline` pipeline, aggregates it, and derives a histogram of `durationInNanos` from the traces based on the value of the `serviceName` field. It then sends the derived metrics to the OpenSearch index called `metrics_for_traces`.
```

**Review by natebower — COMMENTED** [review]
@vagimeli Please see my comments and changes and let me know if you have any questions. Thanks!

File: _data-prepper/common-use-cases/metrics-traces.md (hunk 1 above)
Comment (vagimeli) [reply _data-prepper/common-use-cases/metrics-traces.md:12]: Per @dlvenable Data Prepper doesn't have sub-pipelines, so we can't use that terminology. Revised to read "The pipeline contains the following pipelines:"

**Pull request merged**